examples/api/lib/widgets/basic/custom_multi_child_layout.0.dart

```diff
@@ -0,0 +1,121 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for CustomMultiChildLayout
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const ExampleApp());
+
+class ExampleApp extends StatelessWidget {
+  const ExampleApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Directionality(
+        // TRY THIS: Try changing the direction here and hot-reloading to
```

Comment:
NICE. Drawing the developer's attention to things they ought to try is a nice idea.